lighthouse-core/gather/driver.js

```diff
@@ -187,9 +187,10 @@ class Driver {
    * Call protocol methods
    * @param {!string} method
    * @param {!Object} params
+   * @param {boolean=} silent
    * @return {!Promise}
    */
-  sendCommand(method, params) {
+  sendCommand(method, params, silent) {
```

Comment:
im not crazy about the boolean trap here.  do we need this at all if we follow my idea below?